astro.config.mts

```diff
@@ -0,0 +1,16 @@
+import starlight from '@astrojs/starlight'
+import { defineConfig } from 'astro/config'
+
+// https://astro.build/config
+export default defineConfig({
+  srcDir: './astro',
```

Comment:
This is so that Astro can find its content in the `astro/` folder instead of `src/`

If you prefer a better folder name, feel free to modify it 👌

Reply:
astro is perfect